apps/artwork/components/auction/view.coffee

```diff
@@ -78,7 +78,7 @@ module.exports = class ArtworkAuctionView extends Backbone.View
         width: '500px',
         mode: 'register'
         copy: 'Sign up to bid'
-        destination: form.action()
```

Comment:
You can see this now on https://www.artsy.net/artwork/ella-kruglyanskaya-apples-and-oranges-in-blue?auction_id=phillips-new-now for example.

Click 'bid', sign up, onboarding skipped but you just go back to the homepage.